using-disks.tex

```diff
@@ -0,0 +1,357 @@
+
+The MEGA65 maintains a ``current working directory'' that is used as the base directory for BASIC commands like {\bf MOUNT}. To change the current working directory from BASIC, using the {\bf CHDIR} command with the {\bf U12} argument:
+
+\begin{screenoutput}
+CHDIR "DEMOS",U12
+
+MOUNT "XANADU.D81"
+\end{screenoutput}
+
+
+\section{Using the Internal 3.5" Floppy Disk Drive}
+
+The MEGA65 has a built-in 3.5" floppy disk drive, similar to what was intended for the Commodore 65. You can use physical floppy disks to store your programs and data. Some MEGA65 software can be purchased on floppy disk.
+
+The internal 3.5" drive must be mounted before it can be used. It can be mounted to unit 8 or unit 10.
+
+\subsection{Mounting the 3.5" Drive with the Freezer}
+
+Open the Freezer menu: hold \widekey{RESTORE} for one second, then release it. Notice the current drive mounting settings in the lower-right of the screen.
+
+Press \megakey{0}, then use the cursor down key to: \screentext{- INTERNAL 3.5" -} Press \widekey{RETURN} to select it. The Freezer menu screen shows that the internal drive is mounted to the first managed disk device.
+
+The \screentext{UNIT \#} for the first device can be either 8 or 10. Press \megakey{8} to toggle between these options. BASIC disk commands default to unit 8, so it is typical to use unit 8 unless you are working with multiple disks at the same time.
+
+The internal 3.5" drive cannot be mounted to the second managed drive with unit numbers 9 or 11.
+
+\subsection{Mounting the 3.5" Drive from BASIC}
+
+You can mount the internal 3.5" disk drive to unit 8 using the BASIC {\bf MOUNT} command. This command works from either the \screentext{READY} prompt or from a program. To mount the internal drive to unit 8, enter the command without arguments:
+
+\begin{screenoutput}
+MOUNT
+\end{screenoutput}
+
+The {\bf MOUNT} command can only mount the internal drive to unit 8. You can only mount it to unit 10 from the Freezer menu.
+
+\subsection{DD vs. HD disks}
+
+The MEGA65 disk controller expects a Double Density (DD) floppy disk in the internal 3.5" floppy disk drive.\footnote{It may be possible to support full-capacity HD disks in a future firmware update. This is not a limitation of the drive hardware.} Floppy disks are no longer manufactured, and the DD variety can be difficult to find.
+
+You can use a High Density (HD) floppy disk with the drive, with one important modification: you must cover the hole in the upper-left corner of the disk with a small piece of opaque tape, such as masking tape. This convinces the drive that the disk is DD, and switches it to a mode compatible with the MEGA65 disk controller.
+
+\begin{center}
+  \includegraphics[width=0.7\linewidth]{images/floppy_hd.jpg}
+\end{center}
+
+\underline{Note}: Make sure that the tape is opaque, such as masking tape (not Scotch tape), and that it covers both sides of the hole.
```

Comment:
Recent discussion on Discord showed that the following could be considered:

* The opaqueness of the used tape shouldn't be relevant as all known PC floppy disk drives use mechanical switches.
* The photo just above the line shows a tape covering the upper side of the disk. Usually the switch is reaching partly into the hole from the under side. Covering only the top in such cases is leading the drive to handle the inserted disk still as HD.
* Applying one single piece of tape to cover both holes usually makes the disk wider ever so slightly. Since the force of the eject mechanism is limited disk with such treatment tend to remain stuck. A pair of pliers will in some cases be needed to pull out the disk. It would be better to use two pieces of tape or a much less thicker one than the one being shown on the picture.

Reply:
Fixed the statements about opaqueness, thanks! I've also made sure the text emphasizes covering both sides of the hole. It sounds like a split view photo showing both sides would be useful. The important part is to clearly indicate the location of the hole to cover: an inexperienced user might cover the write-protect hole.

I've never heard of two pieces of tape needing to be used, and would be concerned about too small a piece with too little surface to grip on being more likely to come undone inside the drive. Just interacting with my own MEGA65 it seems like there's plenty of clearance for a single piece of masking tape. (I've heard some people suggest electrical tape which I definitely would not recommend due to thickness and lack of grip.)

(Style note: I feel weird including a photograph in the manual at all, and might consider replacing it with a line drawing if I or someone else can draw one. That would at least avoid a specific implication of the kind of tape to use.)

Reply:
I didn't want to say two pieces are needed. My wording was bad and my intention was only to point out that *if* covering the upper hole had to be shown, covering the underside hole would be the effective action to take. :)

Unfortunately I'm no good in creating vector drawings with the here needed detail being a piece of tape applied to a floppy disk.